app/backend/src/couchers/query.py

```diff
@@ -38,6 +38,17 @@ def filter_by_username_or_id(self, field):
         # no fields match, this will return no rows
         return self.filter(False)
 
+    def filter_by_username_or_email_or_id(self, field):
```

Comment:
Should it just call the other two methods? Or we can refactor out "username" "id" and "email"? Seems kinda weird to have three overlapping methods like this.

Reply:
I can't reuse the other ones because this is doing first some python stuff to pick which `WHERE` to add and then adding that one. So you can't compose the other two to do this.